### PR TITLE
ADD support for power reset of PDU outlet

### PR DESCRIPTION
**Pull request by cxhong**
Add new commands for reset PDU/outlet

```
# rpower f5pdu3 reset
f5pdu3: outlet 1 is reset
f5pdu3: outlet 2 is reset
f5pdu3: outlet 3 is reset
f5pdu3: outlet 4 is reset
f5pdu3: outlet 5 is reset
f5pdu3: outlet 6 is reset
f5pdu3: outlet 7 is reset
f5pdu3: outlet 8 is reset
f5pdu3: outlet 9 is reset
f5pdu3: outlet 10 is reset
f5pdu3: outlet 11 is reset
f5pdu3: outlet 12 is reset

# rpower frame45cn02 pdureset
frame45cn02: f5pdu3 outlet 6 is reset
frame45cn02: f5pdu3 outlet 7 is reset

````

### CODE REVIEW

File: docs/source/advanced/pdu/pdu.rst [hunk 1]
```diff
@@ -63,6 +63,12 @@ The following commands are supported against a compute node:
          cn01: f5pdu3 outlet 6 is on
          cn01: f5pdu3 outlet 7 is on
 
+   * Power cycling the PDU outlets on a compute node: :: 
+   
+       # rpower cn01 pdureset
+         cn01: f5pdu3 outlet 6 is on
+         cn01: f5pdu3 outlet 7 is on
```
Comment (hu-weihua) [inline docs/source/advanced/pdu/pdu.rst]: Should the output look like below

    # rpower cn01 pdureset
    cn01: f5pdu3 outlet 6 is reset
    cn01: f5pdu3 outlet 7 is reset

File: docs/source/advanced/pdu/pdu.rst [hunk 2]
```diff
@@ -112,6 +118,22 @@ The following commands are supported against a PDU:
...
+         f5pdu3: outlet 3 is on
+         f5pdu3: outlet 4 is on
+         f5pdu3: outlet 5 is on
+         f5pdu3: outlet 6 is on
+         f5pdu3: outlet 7 is on
+         f5pdu3: outlet 8 is on
+         f5pdu3: outlet 9 is on
+         f5pdu3: outlet 10 is on
+         f5pdu3: outlet 11 is on
+         f5pdu3: outlet 12 is on
```
Comment (hu-weihua) [inline docs/source/advanced/pdu/pdu.rst]: Should the word ``on`` be ``reset``?

**Comment by hu-weihua** [comment]
@cxhong , I think the code of PDU is fine, what is the openbmc part used for? should we involve other people to review the openbmc part?

**Comment by cxhong** [comment]
@hu-weihua ,  yeh, ok.
when @xuweibj  added openbmc pod file,  she didn't check in the rst files.  so when I ran ```create_man_page.py``` command,  those rst files are generated.   Those files should commit in the PR #2652.

**Comment by xuweibj** [comment]
The codes looks good to me.

**Comment by hu-weihua** [comment]
@cxhong,  you have not modified the part I commented many days ago.  need to change word ``on`` to ``reset`` in the ``pdu.rst``.

**Comment by cxhong** [comment]
@hu-weihua , thanks.   I made changes. please review it.

**Comment by hu-weihua** [comment]
It is good and agree to merge. thanks @cxhong

**Pull request merged**